lib/widgets/countdown_section/countdown_section.dart

```diff
@@ -1,30 +1,97 @@
+import 'package:festival_flutterturkiye_org/core/styles/theme_helper.dart';
+import 'package:festival_flutterturkiye_org/widgets/sign_in_button.dart';
 import 'package:flutter/material.dart';
 
 import 'package:festival_flutterturkiye_org/core/utils/responsive_helper.dart';
 import 'package:festival_flutterturkiye_org/widgets/countdown_section/countdown_exports.dart';
+import 'package:video_player/video_player.dart';
 
 const double _paddingSmall = 24.0;
 const double _paddingMedium = 48.0;
 const double _paddingLarge = 72.0;
 
-class CountdownSection extends StatelessWidget {
-  const CountdownSection();
+class CountdownSection extends StatefulWidget {
```

Comment:
Do we have to use StatefulWidget in the main class?

If it builds again, the whole child widgets will too. And there are some calculations in the child-classes.